src/services/token.services.js

```diff
@@ -0,0 +1,33 @@
+const { Token } = require('../models/token');
+
+const getToken = async (userId) => {
+  const token = await Token.findOne({ where: { userId } });
+
+  return token;
+};
+
+const saveToken = async (userId, refreshToken) => {
+  const token = await getToken(userId);
+
+  if (!token) {
+    const expireAt = new Date(Date.now() + 30 * 24 * 60 * 60 * 1000);
+
+    return Token.create({ userId, refreshToken, expireAt });
+  }
+
+  token.refreshToken = refreshToken;
+  token.expireAt = new Date(Date.now() + 30 * 24 * 60 * 60 * 1000);
```

Comment:
Ensure that the `expireAt` logic aligns with your application's token expiration policy. The current implementation sets the expiration to 30 days from the current date, which should be consistent with the refresh token's validity period.